Cargo.toml

```diff
@@ -77,6 +77,6 @@ test-case = "2.2.2"
 testutils = { path = "lib/testutils" }
 
 [features]
-sapling = ["dag"]
+sapling = ["renderdag"]
```

Comment:
We might want to rename the feature, but this can wait until the crate is published to crates.io.

Reply:
Yeah, i was thinking the same :) We can probably make it an unconditional dependency at that point.